client/src/modules/editor/modules/overview-list/components/OverviewItem.vue

```diff
@@ -0,0 +1,112 @@
+<template>
+    <!-- Nodes related to the label -->
+    <div
+        :class="['node', { selected: isSelected }]"
+        @mousedown="onClick($event)"
+        draggable="true"
+        @dragstart="startDrag($event)"
+    >
+        <p>{{ node.name }}</p>
+        <div class="label" :style="{ background: color, color: fontColor }">
+            {{ node.label }}
+        </div>
+    </div>
+</template>
+
+<script lang="ts">
+import { defineComponent } from "vue";
+import ApiNode from "@/modules/editor/models/ApiNode";
+
+export default defineComponent({
+    name: "OverviewItem",
+    props: {
+        node: {
+            type: Object,
+            default: new ApiNode("", ""),
+        },
+        color: String,
+        fontColor: String,
+    },
+    computed: {
+        /**
+         * @return True if this item is currently selected
+         */
+        isSelected(): boolean {
+            if (!this.$store.state.editor.selectedNode) return false;
+            return this.$store.state.editor.selectedNode.id === this.node.id;
```

Comment:
Maybe delete the first if and replace with: return this.$store.state.editor.selectedNode?.id === this.node.id;